services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx

```diff
@@ -15,27 +15,43 @@ along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>
 Copyright 2023 freiheit.com*/
 import './ProductVersion.scss';
 import * as React from 'react';
-import { refreshTags, useTags, getSummary, useSummaryDisplay } from '../../utils/store';
+import { refreshTags, useTags, getSummary, useSummaryDisplay, useEnvironmentGroups } from '../../utils/store';
 import { DisplayManifestLink, DisplaySourceLink } from '../../utils/Links';
 import { Spinner } from '../Spinner/Spinner';
 import { ProductSummary } from '../../../api/api';
+import { useSearchParams } from 'react-router-dom';
 
-export type ProductVersionProps = {
-    environment: string;
+// splits up a string like "dev:dev-de" into ["dev", "dev-de"]
+const splitCombinedGroupName = (envName: string): string[] => {
+    const splitter = envName.split('/');
+    if (splitter.length === 1) {
+        return ['', splitter[0]];
+    }
+    return [splitter[1], ''];
 };
 
-export const ProductVersion: React.FC<ProductVersionProps> = (props) => {
+export const ProductVersion: React.FC = () => {
     React.useEffect(() => {
         setShowTagsSpinner(true);
         refreshTags();
     }, []);
-    const { environment } = props;
+    const envGroupResponse = useEnvironmentGroups();
+    const envList: string[] = [];
+    for (let i = 0; i < envGroupResponse.length; i++) {
+        envList.push(envGroupResponse[i].environmentGroupName);
+        for (let j = 0; j < envGroupResponse[i].environments.length; j++) {
+            envList.push(envGroupResponse[i].environmentGroupName + '/' + envGroupResponse[i].environments[j].name);
+        }
+    }
```

Comment:
nitpick: It would be nice to extract this into a seperate hook function, e.g. `useEnvironmentGroupCombinations"